docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+db:
+  image: postgres:9.4
+  ports:
+  - "5432:5432"
+web:  
+  build: .
+  command: bundle exec puma -C config/puma.rb
+  ports:
+  - "3000:3000"
+  volumes:
+   - .:/app
+  links:
+  - db
+  environment:
+    DB_HOST: db
+    DB_USER: postgres
```

Comment:
Same deal here; it's likely keeping these uniform across the board if we can help it https://github.com/artsy/gris/pull/73